### ISSUE TITLE
Saar multiplier

### ISSUE THREAD

**Issue by DerekYves** [christophsax/seasonal#215]
Hello, thank you for this great package. I'm working with a data set that is different from but substantively similar to the US census permit survey for new residential construction. What I'd like to do is calculate an analogue for the "SAAR" multiplier the Census publishes each month with its new report. This is the single value they report which scales an annualized raw permit count to a SAAR. I understand they create this multiplier with X13. Are there an examples of this kind worked out that you can direct me toward? Would I first annualize my monthly raw estimate, perform the seasonal adjustment, and back in to the monthly multiplier for each month in the series by comparing the raw and adjusted series? Any tips greatly appreciated.

**Comment by christophsax** [comment]
No experience with SAAR, just had to google it. It seems SAAR is just the seasonally adjusted series times 12 (monthly) or 4 (quarterly)?

**Comment by DerekYves** [comment]
Hi Chris,

In my case it is a monthly series. You may have seen this:

https://www.census.gov/construction/bps/faqs/faqs_seas.html

The appear, for the many housing surveys the publish, to be simply dividing
a given month' raw estimate by the seasonal factory hat is calculated for a
two year window of data. What I can't find from them is any description of
the actual calls to x13 that produce the factor itself.  Many thanks for
responding and for any tips you might have.

**Comment by christophsax** [comment]
Getting the seasonal factor is easy:

``` r
library(seasonal)
m <- seas(AirPassengers, x11 = "")
series(m, "seats.adjustfac")
```
Also, see `?series` for how to access any series of X-13.

**Comment by DerekYves** [comment]
Thank you so much for your quick response.  For anyone who finds this thread in the future with a similar question, I was able to roughly approximate the U.S. Census seasonal factor for new housing permits when using the most recent three years of thir series. It does appear they are using some kind of a future forecast in there Model as well, which perhaps accounts for some of the differences I did see.
